templates/test/ci/cluster-template-prow-aks-multi-tenancy.yaml

```diff
@@ -70,7 +70,7 @@ metadata:
   name: agentpool0
   namespace: default
 spec:
-  maxPods: 24
+  maxPods: 30
```

Comment:
verify test failed because you need to modify the kustomize patch and then run make generate instead of changing the generated template directly (or you can cheat and manually change both as long as the generated output produces no diff)

Reply:
🤦 will fix it now